checks/branch_protection_test.go

```diff
@@ -348,7 +348,7 @@ func TestReleaseAndDevBranchProtected(t *testing.T) {
 		{
 			name: "TargetCommittish nil",
 			expected: scut.TestReturn{
-				Errors:        []error{sce.ErrScorecardInternal},
+				Error:         sce.ErrScorecardInternal,
```

Comment:
note: the list of errors was intentional: any time we have a wrapped error we expose (currently none :-)), we could check it using the list.